### DISCUSSION TITLE
Manuals/Books on Solidity (Recommendations)

### DISCUSSION THREAD

**Discussion by purplestoke** [smartcontractkit/full-blockchain-solidity-course-py#1854] [Q&A] [answered]
Just asking around for any good books on Solidity. I find it easier to learn that way but it seems like there aren't to many options out there. Preferably something based around Ethereum that takes EIP-1559 into account. Or if someone knows a good manual that really breaks down the basics of Solidity. 

cheers 👾

**Comment by cromewar** [comment, marked as answer]
Hello @purplestoke here you have some recommendations:

- Mastering Blockchain Programming with Solidity by Jitendra Chittoda - This book provides a comprehensive introduction to the Solidity language, covering everything from basic syntax to advanced features.
- Solidity Programming Essentials by Ritesh Modi - This book is a great introduction to Solidity programming, covering the basics of the language and smart contract development.
- Building Ethereum DApps: Decentralized Applications on the Ethereum Blockchain by Roberto Infante - This book provides a practical guide to building decentralized applications using Solidity and other Ethereum tools.
- Ethereum Smart Contract Development: Build blockchain-based decentralized applications using Solidity by Mayukh Mukhopadhyay - This book is a great resource for developers who want to learn how to build decentralized applications using Solidity and Ethereum.
"Hands-On Blockchain for Python Developers: Gain blockchain programming skills to build decentralized applications using Python" by Arjuna Sky Kok
